.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,22 @@
+name: Pull Request Action
+on:
+  push:
+    branches:
+      - feat/bot-*
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    env:
+      BRANCH_NAME: ${{ github.ref_name }}
```

Comment:
would be better to use `${{ github.ref_name }}`  in the actual body? then I dont have to go and look what these are